cdc/scheduler/balancer.go

```diff
@@ -127,9 +155,22 @@ func (r *tableNumberBalancer) FindVictims(
 		for tableID := range tables {
 			tableList = append(tableList, tableID)
 		}
-		// We sort the tableIDs here so that the result is deterministic,
-		// which would aid testing and debugging.
-		util.SortTableIDs(tableList)
+
+		if r.random != nil {
```

Comment:
In which scenario will `r.random` be nil?

Reply:
When the balancer has been created with `newDeterministicTableNumberRebalancer`.

Reply:
Yep, I found `newDeterministicTableNumberRebalancer` is only used in unit test, will we really use it in the future.

Reply:
I don't have a plan to use it for now. It is mainly for testing purpose.